spec/spec_helper_acceptance.rb

```diff
@@ -2,6 +2,27 @@
 require 'mustache'
 require 'open3'
 
+unless ENV['DEVELOPMENT']
+  require 'beaker-rspec'
+  require 'beaker/puppet_install_helper'
+  unless ENV['BEAKER_provision'] == 'no'
+    install_pe
+    on(default, puppet('module install puppetlabs-aws'))
+
+    # install the AWS SKD & retries gems on the agent
+    on(default, puppet('resource package aws-sdk-core ensure=installed provider=puppet_gem'))
+    on(default, puppet('resource package retries ensure=installed provider=puppet_gem'))
+
+    # install aws creds on SUT
+    home = ENV['HOME']
+    file = File.open("#{home}/.aws/credentials")
+    agent_home = on(default, 'printenv HOME').stdout.chomp
+    on(default, "mkdir #{agent_home}/.aws")
+    create_remote_file(default, "#{agent_home}/.aws/credentials", file.read)
+    # TODO enable promotion of artifact to modules forge
```

Comment:
That would be preferable, especially for external users of this. What's the relevant beaker incantation?